plugins/fileservices/fileservices.cpp

```diff
@@ -2248,6 +2250,15 @@ FILESERVICES_API char * FILESERVICES_CALL fsfRemotePull_impl(ICodeContext *ctx,
 
     req->setExpireDays(expireDays);
 
+    // Store username/psw
```

Comment:
perhaps a bit misleading to say 'Store' here, it mighy imply it is permanent and a security issue, whereas really it's just setting into an object used by the request.

Reply:
Changed